install/cupy_builder/install_build.py

```diff
@@ -228,7 +231,7 @@ def _match_output_lines(output_lines, regexs):
     return None
 
 
-def get_compiler_base_options(compiler_path):
+def get_compiler_base_options(compiler_path) -> List[str]:
```

Comment:
Thanks, but surprisingly this is a list 😇 

```suggestion
def get_compiler_base_options(compiler_path: List[str]) -> List[str]:
```

(That's why we need a refactoring in installer...)